src/quadpype/hosts/blender/api/lib.py

```diff
@@ -572,3 +579,41 @@ def get_cache_modifiers(obj, modifier_type="MESH_SEQUENCE_CACHE"):
                                    if modifier.type == modifier_type]
                 modifiers_dict[ob.name] = cache_modifiers
     return modifiers_dict
+
+
+def get_parents_for_collection(collection, collections=None):
+    if not collections:
+        collections = bpy.data.collections
+    return [c for c in collections if c.user_of_id(collection)]
+
+
+def get_asset_children(asset):
+    return list(asset.objects) if isinstance(asset, bpy.types.Collection) else list(asset.children)
+
+
+def get_and_select_camera(objects):
+    for blender_object in objects:
+        if blender_object.type == "CAMERA":
+            blender_object.select_set(True)
+            return blender_object.data
+
+        camera = get_and_select_camera(list(blender_object.children))
+        if camera:
+            return camera
+
+
+def extract_sequence_and_shot():
+    # TODO : Needs to be improved (not reliable at this time)
+    return get_current_context()['asset_name'].split('_')
+
+
+def is_camera(obj):
+    return isinstance(obj, bpy.types.Object) and obj.type == "CAMERA"
+
+
+def is_collection(obj):
+    return isinstance(obj, bpy.types.Collection)
+
+
+def is_shot():
```

Comment:
Here also a `TODO` need to be added just like for the `extract_sequence_and_shot` 

And maybe use `return len(extract_sequence_and_shot()) > 1` like that there is only one place where the "hardcoded" logic is used